cli.js

```diff
@@ -34,7 +34,7 @@ function action(paths, options) {
         fs.accessSync(element, fs.F_OK);
         return true;
     } catch (e) {
-      console.log("Warning: File or directory \"" + element + "\" does not exist.");
+      console.log('Warning: File or directory \'' + element + '\' does not exist.');
```

Comment:
Why not `console.log('Warning: File or directory "' + element + '" does not exist.');`